src/pkg/ocidriver/ociimage.go

```diff
@@ -0,0 +1,22 @@
+package ocidriver
+
+import (
+	"github.com/opencontainers/go-digest"
+)
+
+// OciImage ...
+type OciImage struct {
```

Comment:
There is no resource called `image` in the registry API, only `manifest` and `blob`